debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java

```diff
@@ -115,12 +115,28 @@ public ValueConverter converter(Column column, Field fieldDefn) {
                 return data -> convertPoint(column, fieldDefn, data);
             case PgOid.MONEY:
                 return data -> convertMoney(column, fieldDefn, data);
+            case PgOid.NUMERIC:
+                return data -> convertDecimal(column, fieldDefn, data);
             default:
                 return super.converter(column, fieldDefn);
         }
     }
     
     @Override
+    protected Object convertDecimal(Column column, Field fieldDefn, Object data) {
+        BigDecimal newDecimal = (BigDecimal) super.convertDecimal(column, fieldDefn, data);
+        if (newDecimal == null) {
+            return newDecimal;
+        }
+        if (column.scale() > -1) {
+            String decimalWithTrailingZeros = String.format("%-" + column.scale() + "s", newDecimal.doubleValue())
```

Comment:
I found that simply changing the scale does not affect the string serialized value of the BigDecimal, so it does not recover the missing trailing zeros.
Anyway I found maybe a more elegant solution using the `MathContext` instead of the string:

    MathContext mc = new MathContext((newDecimal.precision() - newDecimal.scale()) + column.scale());
    newDecimal = new BigDecimal(newDecimal.doubleValue(), mc);

So in the case of the integration test, where I have a DECIMAL(3,2) and a database value of 1.1:

`newDecimal.precision()` assumes the value of `2`
`newDecimal.scale()` assumes the **wrong** value of `1`
`newDecimal.precision() - newDecimal.scale()` is the size of the integer part of the original newDecimal (so it would assume the value `1`)
`column.scale()` is the **correct scale** for the newDecimal, so it would be `2`

The MathContext would consider a precision of `3` and the value will be correctly represented as `1.10`.